fitbenchmarking/results_processing/local_min_table.py

```diff
@@ -90,17 +90,16 @@ def get_value(self, result):
                  specified above) and :math:`\\frac{|| J^T r||}{||r||}`
         :rtype: bool, float
         """
-
-        res = result.r_x
-
-        jac = result.jac_x
-
-        if res is None:
+        if result.accuracy == float('inf'):
```

Comment:
Quick question on why you don't choose numpy for this? I didn't realise you could do float('inf').

```suggestion
        if np.isinf(result.accuracy):
```

Reply:
This was just a way to avoid the import and see if I could do it without adding another dependency into the file. But either way is fine to me. I will update these two checks to use np.inf instead.